bavi/modules/random.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from random import choice
+
+log = logging.getLogger('bavi.modules.random')
+
+supported_split = [',', '|', '/', '\\']
+
+RESPONSE_MESSAGE = "Your choices: {0}, I chose: {1}."
```

Comment:
I forgot to document it under `CONTRIBUTING.md`, but the convention I've adopted is using single quotes for strings unless the string contains a literal quote.

Also since your "no choices" case has the constant inline, I think it would be fine to have this one be inline as well.